### PR TITLE
Lower minSdk to API 14

### PR DESCRIPTION
**Pull request by hzsweers**
None of the changes from the last release necessitate 15 over 14, and there's virtually no relevant API differences between 14 and 15 that would realistically affect this project in the future. Many developers do however have min API versions of 14, and would either have to set up their manifest to ignore Facebook's minSDK version or drop the SDK all together.

To use us as an example: We have nearly 1million installs on API 14 devices.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ANDROID_BUILD_MIN_SDK_VERSION=15
+ANDROID_BUILD_MIN_SDK_VERSION=14
```
Comment (PopsBGone69) [inline gradle.properties]: Peace A like more information about the same time 😊

**Comment by swiese** [comment]
Hzsweers you are correct that none of the apis that we use require api 15. The reason we made the move to api 15 instead of api 14 was to remain inline with Facebook and Messenger app for Android which have a minimum api of 15. 

When we made this change we made sure to research the number of users on 4.0.0, 4.0.1, and 4.0.2 and found it to be an insignificant number of users. The Google dashboard further confirms our data that this api level has a low number of active users. 

https://developer.android.com/about/dashboards/index.html. 
Note that API level 14 isn't even displayed in the chart since its reach is under 0.05%. 

I would suggest that you look into the number of active users your app has on API 14 and make sure it's significant. It could be that for your specific app this api version is popular and I'm glad there is a workaround for your team in ignoring the min sdk version.

Thanks for bring up this issue and I hope this clarifies why we made the decision to set the min sdk to 15.

**Comment by hzsweers** [comment]
I understand the decision, but this SDK isn't just used in Facebook's apps.
Relatively speaking, yes 4.0-4.0.2 devices are rare. We're bundled on
millions of Samsung devices, which includes a lot of users that are stuck
with devices that Samsung didn't update. I don't quite see the harm in
supporting 14 when there's no cost to the library beyond an artificial
measure of minimum SDK parity.

**Comment by hzsweers** [comment]
I understand the decision is final though, and we'll figure something out
on our end.

**Pull request closed without merging**